tests/api.c

```diff
@@ -5376,9 +5386,21 @@ static int test_client_nofail(void* args, cbType cb)
         goto done;
     }
 
-    if (wolfSSL_set_fd(ssl, sockfd) != WOLFSSL_SUCCESS) {
-        /*err_sys("SSL_set_fd failed");*/
+    if (!doUdp) {
+        if (wolfSSL_set_fd(ssl, sockfd) != WOLFSSL_SUCCESS) {
+            /*err_sys("SSL_set_fd failed");*/
```

Comment:
why these were commented out? should we replace them with a normal comment?

Reply:
I think this was done to return instead of using `exit()` or `abort()`. I think this was part of @haydenroche5's refactor of the tests.

Reply:
In my refactor, I didn't actually modify any test behavior. Most tests are still using asserts, which abort. So, not sure what the story is behind this commented out err_sys.

Reply:
Appears it was this way from the very start in https://github.com/wolfSSL/wolfssl/commit/8c63ac1c4d